Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -8039,6 +8039,17 @@ This class is used by plugins wishing to display a custom window to the player,
 					},
 					Notes = "Returns the cItemGrid object representing the internal storage in this window",
 				},
+				SetOnClicked =
+				{
+					Params =
+					{
+						{
+							Name = "OnClickedCallback",
+							Type = "function",
+						},
+					},
+					Notes = "Sets the function that the window will call when it is about to process a click from a player. See below for the signature of the callback function.",
```

Comment:
I couldn't find any prior examples for how to make a link to a section, but this worked (tested with APIDump and a browser).